src/app/pages/details/desktop-view/desktop-view.js

```diff
@@ -5,23 +5,41 @@ import DetailsTab from './details-tab/details-tab';
 import InstructorResourceTab from './instructor-resource-tab/instructor-resource-tab';
 import StudentResourceTab from './student-resource-tab/student-resource-tab';
 import VideoTab from './videos-tab/videos-tab';
+import useLanguageContext from '~/models/language-context';
 import useDetailsContext from '../context';
 import {GiveLink} from '../common/common';
 import $ from '~/helpers/$';
 import './desktop-view.scss';
 
+const tabsByLanguage = {
+    en: [
+        'Book details',
+        'Instructor resources',
+        'Student resources',
+        'Videos'
+    ],
+    es: [
+        'Detalles del libro',
+        'Recursos del instructor',
+        'Recursos para estudiantes',
+        'Videos'
+    ]
+};
```

Comment:
if this starts to become common, we should move all the strings into a json/yaml file where translators could work on them

Reply:
I peppered a lot of translations throughout the code as a stopgap. Ultimately, most of them should be added to the CMS and removed from the FE.